recipes/hmat-oss/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+patch -p1 -i ${RECIPE_DIR}/7d14b6451d3f63a949fee9c09d349aec6edd193e.patch
+patch -p1 -i ${RECIPE_DIR}/2bea755a2139907d331bfa0229154b6dab7a61c1.patch
+patch -p1 -i ${RECIPE_DIR}/aad9de409bc2e0a8a50fed1169df0a06a252f399.patch
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBLAS_LIBRARIES=${PREFIX}/lib/libopenblas${SO_EXT} \
+  -DBUILD_EXAMPLES=ON \
+  -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib" \
+  -DCMAKE_MACOSX_RPATH=ON \
+  -DCMAKE_CXX_FLAGS="-std=c++11" \
+  ..
+
+make install
+rm -r ${PREFIX}/bin/examples
+./c-simple-cylinder 1000 D
```

Comment:
@jakirkham it solve a finite-element problem over a cylindric mesh, seems to test various aspects of the lib